authentication/pipeline/user.py

```diff
@@ -90,6 +91,7 @@ def require_password_and_profile_via_email(
     if "name" in data:
         profile.name = data["name"]
         profile.save()
+    index_new_profile(profile)
```

Comment:
Instead of putting calling `index_new_profile` here and below, can we add a signal so it gets called every time `profile.save()` happens?

Reply:
The `Profile` model already has a custom `save` function, how about I put it there?